### PR TITLE
Use logfmt as the default output for influxdb logging

### PR DESCRIPTION
**Pull request by jsternberg**

### CODE REVIEW

File: cmd/influxd/main.go [hunk 1]
```diff
@@ -61,7 +62,7 @@ type Main struct {
 func NewMain() *Main {
 	return &Main{
 		Logger: zap.New(
-			zap.NewTextEncoder(),
+			zaplogfmt.NewEncoder(),
```
Comment (e-dard) [inline cmd/influxd/main.go]: What about all the other places we're currently using `zap.NewTextEncoder()`? Do we need to replace those too?

Comment (jsternberg) [inline cmd/influxd/main.go]: Nope. But I'll modify the one in `cmd/influxd/run/server.go` anyway. The rest are in test files.

**Comment by jwilder** [comment]
The current `zap.TextEncoder` is pretty readable.  I don't think we need to change the format again.

**Comment by jsternberg** [comment]
Closing this then.

**Pull request closed without merging**